recipes/netket/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "netket" %}
+{% set version = "2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/netket/netket/archive/v2.1.tar.gz
+  sha256: 041d2b058c5d2945bf0a4194ad2bf3c426ad9b6ce0dd323a81a7154bc6e45452
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv --install-option=--cmake-args='-DNETKET_USE_LAPACK=OFF -DCMAKE_BUILD_TYPE=Release -DNETKET_USE_OPENMP=ON -DNETKET_USE_BLAS=ON -DNETKET_USE_SLEEF=ON'"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake >=3.12
+    - ninja
+  host:
+    - pip
+    - python =3.7
```

Comment:
See https://conda-forge.org/docs/maintainer/adding_pkgs.html#skipping-builds on how to skip python versions